kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF-AI/mii-initial-d1-JRF-AI-v1.yaml

```diff
@@ -147,8 +133,10 @@ spec:
     model:
       # Valid model domain types are 'WLS', 'JRF', and 'RestrictedJRF', default is 'WLS'
       domainType: "JRF"
-      modelHome: "/auxiliary/models"
-      wdtInstallHome: "/auxiliary/weblogic-deploy"
+
+      # Auxiliary image(s) containing WDT model, archives and install.
+      auxiliaryImages:
+      - image: "model-in-image:JRF-AI-v1"
```

Comment:
>>    #    Files are copied from `sourceModelHome` to `spec.configuration.model.modelHome`
>>   #    and from `sourceWDTInstallHome` to `spec.configuration.wdtInstallHome`.
[ANIL] I think above comment could be incorrect if user explicitly configures `spec.configuration.model.modelHome` or `spec.configuration.wdtInstallHome` to a non-default value.

Reply:
Good point.
```
    #    Optional auxiliary image(s) containing WDT model, archives, and install.
    #    Files are copied from `sourceModelHome` to the `models` directory
    #    in the `spec.configuration.model.auxiliaryImageVolumeMountPath` (default `/aux`).
    #    Similarly, files are copied from `sourceWDTInstallHome` to the `weblogic-deploy` directory under `auxiliaryImageVolumeMountPath`.
    #    If you change `auxiliaryImageVomeMountPath`, then you will likely
    #    also want to customize `spec.configuration.model.modelHome` and
    #    `spec.configuration.wdtInstallHome` so that they correspond to these
    #    target directories.
    #      `image`                - Image location
    #      `imagePullPolicy`      - Pull policy, default `IfNotPresent`
    #      `sourceModelHome`      - Model file directory in image, default `/auxiliary/models`.
    #      `sourceWDTInstallHome` - WDT install directory in image, default `/auxiliary/weblogic-deploy`.
```

Reply:
Based on latest slacks, now looking for something like:
```
    #    Optional auxiliary image(s) containing WDT model, archives, and install.
    #    Files are copied from `sourceModelHome` in the aux image to the `/aux/models` directory
    #    in running WebLogic Server pods, and files are copied from `sourceWDTInstallHome` 
    #    to the `/aux/weblogic-deploy` directory. Set `sourceModelHome` and
    #.   or `sourceWDTInstallHome` to "None" if you want skip such copies.
    #      `image`                - Image location
    #      `imagePullPolicy`      - Pull policy, default `IfNotPresent`
    #      `sourceModelHome`      - Model file directory in image, default `/auxiliary/models`.
    #      `sourceWDTInstallHome` - WDT install directory in image, default `/auxiliary/weblogic-deploy`.
```